.github/workflows/tx-hammer-plugins.yml

```diff
@@ -0,0 +1,96 @@
+name: Translations
+
+on:
+  workflow_call:
+    inputs:
+      gem_name:
+        description: 'Name of the Gem'
+        required: true
+        type: 'string'
+    secrets:
+      tx_token:
+        description: 'API token for the Transifex.'
+        required: true
+      gh_token:
+        description: 'GitHub token with permissions to create a pull request'
+        required: true
```

Comment:
I'd think this isn't needed. GitHub already has the right permissions if it runs as an action.